### DISCUSSION TITLE
SDRPP issue in linux mint 21

### DISCUSSION THREAD

**Discussion by Metalblade66** [AlexandreRouma/SDRPlusPlus#1057] [General]
I downloaded the file sdrpp_debian_sid_amd64.deb, then run it and so on, but at the moment to open the sdrpp from a terminal it showed the the file libvolk.so.2.4 doesn`t exist, but instead of it the systeam has the file libvolk.so.2.5, so I used sudo ln -sf /usr/lib/x86_64-linux-gnu/libvolk.so.2.5 /usr/lib/x86_64-linux-gnu/libvolk.so.2.4, and the problem was solved and now I can open the software, but when I played the software crash and close without any warning or error on console, can you help to know what`s going on.

I am using a Dell Inspiron 14 series 3000, with a SSD and 8 GB RAM, OS Linux mint 21 codename: vanessa

**Comment by AlexandreRouma** [comment]
How is it surprising that a deb made for debian sid doesn't have the right library versions for mint 21?
Download the the for the distro that mint 21 is based on, aka ubuntu jammy.
